### PR TITLE
add `eventID` to `createPost` mutation

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: application/gqlgen/schema.graphql [hunk 1]
```diff
@@ -211,6 +211,7 @@ input CreatePostInput {
     url: String
     kilograms: Float
     photoID: ID
+    eventID: ID
```
Comment (Baggerone) [inline application/gqlgen/schema.graphql]: This should be meetingID, right?

File: application/gqlgen/post.go [hunk 2]
```diff
@@ -311,18 +321,18 @@ func (r *mutationResolver) CreatePost(ctx context.Context, input postInput) (*mo
 	}
 
 	dest := convertGqlLocationInputToDBLocation(*input.Destination)
-	if err1 := dest.Create(); err1 != nil {
-		return nil, reportError(ctx, err1, "CreatePost.SetDestination", extras)
+	if err = dest.Create(); err != nil {
```
Comment (Baggerone) [inline application/gqlgen/post.go]: Thanks for changing  these.

File: application/actions/post_test.go [hunk 3]
```diff
@@ -235,6 +236,7 @@ func (as *ActionSuite) Test_CreatePost() {
 
 	as.Equal(f.Organization.UUID.String(), postsResp.Post.Organization.ID)
 	as.Equal(f.File.UUID.String(), postsResp.Post.Photo.ID)
+	//as.Equal(f.Meetings[0].UUID.String(), postsResp.Post.Meeting.ID)
```
Comment (Baggerone) [inline application/actions/post_test.go]: Should this still be commented out?

File: application/gqlgen/post.go [hunk 4]
```diff
@@ -281,6 +282,14 @@ func convertGqlPostInputToDBPost(ctx context.Context, input postInput, currentUs
 		}
 	}
 
+	if input.EventID != nil {
```
Comment (Baggerone) [inline application/gqlgen/post.go]: (MeetingID?)

**Comment by Schparky** [comment]
> What the UI exposes to the user can use "event", but the gql schema should still be based on the back end terminology, right?

I guess you're right. I did find it confusing and I got it wrong a few times.

File: application/actions/post_test.go (hunk 3 above)
Comment (Schparky) [reply application/actions/post_test.go]: Yes, until I get your schema update.

**Pull request merged**